assets/src/dashboard/animations/parts/test/index.js

```diff
@@ -65,4 +65,19 @@ describe('AnimationPart', () => {
       expect(getByTestId('child-rendered')).toBeDefined();
     }
   );
+
+  it('should return objects with the same signature for all parts and throughput', () => {
+    // throughput should have the same signature as all parts
+    const throughputNames = Array.from(Object.keys(throughput())).sort();
+
+    Object.values(ANIMATION_TYPES).forEach((type) => {
+      const properties = Array.from(
+        Object.keys(AnimationPart(type, {}))
+      ).sort();
+
+      // 'type' is being added to the assertion so we'll
+      // know which animation part was being tested
+      expect({ [type]: properties }).toStrictEqual({ [type]: throughputNames });
```

Comment:
I notice that I forgot to update `throughput` with the appropriate return types (specifically I didn't rename the AMP props to the new names of AMPTarget and AMPAnimation) so I wrote this unit test to make sure we won't fudge names again.  `throughput` should be the standard of truth!